integration-tests/distributioncheck/go.mod

```diff
@@ -1,28 +1,28 @@
 module distributioncheck
 
 require (
-	github.com/aws/aws-xray-sdk-go latest
+	github.com/aws/aws-xray-sdk-go v1.7.1
```

Comment:
Hmm let's keep this to `latest`? That way we don't need to remember to update it every time we cut a new release.